### PR TITLE
Add missing Unit tests.

### PR DESCRIPTION
**Pull request by drupol**
Container::camelize() and Container::underscore() are now covered.

### CODE REVIEW

File: tests/src/DependencyInjection/ContainerTest.php [hunk 1]
```diff
@@ -385,6 +385,25 @@ public function test_initialized() {
     $this->assertTrue($this->container->initialized('late.service'), 'Late service is initialized after it was gotten.');
   }
 
+  /**
+   * Camelizes a string.
+   *
+   * @covers ::camelize()
```
Comment (dawehner) [inline tests/src/DependencyInjection/ContainerTest.php]: don't use () at the end ...

File: tests/src/DependencyInjection/ContainerTest.php [hunk 2]
```diff
@@ -385,6 +385,25 @@ public function test_initialized() {
...
+   */
+  public function test_camelize() {
+    $result = $this->container->camelize('service_container');
+    $this->assertEquals('ServiceContainer', $result, 'The strings are the same.');
+  }
+
+  /**
+   * Un-camelizes a string.
+   *
+   * @covers ::underscore
```
Comment (fabianx-ai) [inline tests/src/DependencyInjection/ContainerTest.php:402]: I would like to use data providers for that and test a little more scenarios.

Note:

The same data provider can be used here for both functions.

File: tests/src/DependencyInjection/ContainerTest.php [hunk 3]
```diff
@@ -385,6 +385,41 @@ public function test_initialized() {
...
 
+  /**
+   * Camelizes a string.
+   *
+   * @covers ::camelize
+   * @dataProvider underscoreCamelizeDataProvider
+   */
+  public function test_camelize($string_underscore, $string_camelize) {
+    $result = $this->container->camelize($string_underscore);
+    $this->assertEquals($string_camelize, $result, 'The strings are the same.');
```
Comment (fabianx-ai) [inline tests/src/DependencyInjection/ContainerTest.php]: Lets remove the 'The strings are the same.' here - it does not give new information.

File: tests/src/DependencyInjection/ContainerTest.php [hunk 4]
```diff
@@ -385,6 +385,41 @@ public function test_initialized() {
...
+
+  /**
+   * Un-camelizes a string.
+   *
+   * @covers ::underscore
+   * @dataProvider underscoreCamelizeDataProvider
+   */
+  public function test_underscore($string_underscore, $string_camelize) {
+    $result = $this->container->underscore($string_camelize);
+    $this->assertEquals($string_underscore, $result, 'The strings are the same.');
```
Comment (fabianx-ai) [inline tests/src/DependencyInjection/ContainerTest.php]: Lets remove the 'The strings are the same.' here - it does not give new information.

**Comment by fabianx-ai** [comment]
Looks perfect!

**Pull request merged**